lib-py/casmutils/xtal/lattice.py

```diff
@@ -1,15 +1,242 @@
 from . import _xtal
+from . import globaldef
 
-class _Lattice(_xtal.Lattice):
+class Equals:
+
+    """A wrapper class for _xtal.LatticeEquals_f"""
```

Comment:
This is information about the implementation. A user that reads this will have no idea what you're talking about.